tests/fixture/db.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
We should remove these actually as they won't be needed anymore.

Reply:
Sure, I should probably make a fork of the template repo and just stick some fixtures in there so we can spin up a version and add in all the relevant resources that we will need to be testing as this develops. Is that the method you'd recommend?